MobalMobal/MobalMobal/Main/cell/MainOngoingDonationTableViewCell.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MainOngoingDonationTableViewCell.swift
+//  MobalMobal
+//
+//  Created by 김재희 on 2021/02/28.
+//
+
+import UIKit
+
+class MainOngoingDonationTableViewCell: UITableViewCell {
+    // MARK: - UIComponent
+    let collectionView: UICollectionView = {
+        let collectionView: UICollectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+        let layout: UICollectionViewFlowLayout = UICollectionViewFlowLayout()
+        collectionView.collectionViewLayout = layout
```

Comment:
여기도 layout을 먼저 선언하는 방법으로 조심스레 제안해봅니다!
```suggestion
        let layout: UICollectionViewFlowLayout = UICollectionViewFlowLayout()
        let collectionView: UICollectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
```